docs/pages/kubernetes-access/guides/cicd.mdx

```diff
@@ -35,33 +36,36 @@ spec:
   - robot
 ```
 
-Generate a kubeconfig using the `jenkins` user and its roles using [`tctl auth sign`](../../cli-docs.mdx#tctl-auth-sign):
+<Admonition type="warning" title="Warning">
+  The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in customer-facing production environments per the *Principle of Least Privilege* (POLP). 
+</Admonition>
+
+Generate a `kubeconfig` using the `jenkins` user and its roles using [`tctl auth sign`](../../cli-docs.mdx#tctl-auth-sign):
 
 ```bash
 # Create a new local user for Jenkins
-$ tctl users add jenkins --roles=robot
+sudo tctl users add jenkins --roles=robot
 # Creates a token for 25hrs
-$ tctl auth sign --user=jenkins --format=kubernetes --out=kubeconfig --ttl=25h
+sudo tctl auth sign --user=jenkins --format=kubernetes --out=kubeconfig --ttl=25h
 
-  The credentials have been written to kubeconfig
+#  The credentials have been written to kubeconfig
 
-$ cat kubeconfig
-  apiVersion: v1
-  clusters:
-  - cluster:
-      certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZ....
+cat kubeconfig
+#   apiVersion: v1
```

Comment:
I left the cat command commented out since it's incomplete. Happy to uncomment this.